convert.py

```diff
@@ -0,0 +1,52 @@
+import binascii
```

Comment:
Something like in tlsfuzzer ? And copy the LICENSE file also ?

# Author: Hubert Kario
# Released under Gnu GPL v2.0, see LICENSE file for details

Though you want to do this.